fastlane/Fastfile

```diff
@@ -97,402 +95,408 @@ lane :test do | options |
-    new_version = ios_get_app_version(public_version_xcconfig_file: VERSION_FILE_PATH)
-    ios_update_release_notes(new_version: new_version)
-    setbranchprotection(repository:GHHELPER_REPO, branch: "release/#{new_version}")
-    setfrozentag(repository:GHHELPER_REPO, milestone: new_version)
-
-    generate_strings_file_for_glotpress()
-  end
+#####################################################################################
+# code_freeze
+# -----------------------------------------------------------------------------------
+# This lane executes the steps planned on code freeze
+# -----------------------------------------------------------------------------------
+# Usage:
+# bundle exec fastlane code_freeze [skip_confirm:<skip confirm>]
+#
+# Example:
+# bundle exec fastlane code_freeze
+# bundle exec fastlane code_freeze skip_confirm:true
+#####################################################################################
+desc 'Creates a new release branch from the current trunk'
+lane :code_freeze do |options|
+  ios_codefreeze_prechecks(options)
+
+  ios_bump_version_release
+  new_version = ios_get_app_version(public_version_xcconfig_file: VERSION_FILE_PATH)
+  ios_update_release_notes(new_version: new_version)
+  copy_branch_protection(
+    repository: GITHUB_REPO,
+    from_branch: DEFAULT_BRANCH,
+    to_branch: "release/#{new_version}"
+  )
+  setfrozentag(
+    repository: GITHUB_REPO,
+    milestone: new_version
+  )
 
-  #####################################################################################
-  # update_appstore_strings
-  # -----------------------------------------------------------------------------------
-  # This lane updates the AppStoreStrings.pot files with the latest content from
-  # the release_notes.txt file and the other text sources
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  # bundle exec fastlane update_appstore_strings version:<release note version>
-  #
-  # Example:
-  # bundle exec fastlane update_appstore_strings version:1.1
-  #####################################################################################
-  desc "Updates the AppStoreStrings.pot file with the latest data"
-  lane :update_appstore_strings do | options |
-    source_metadata_folder = File.join(PROJECT_FOLDER, "fastlane/appstoreres/metadata/source")
-
-    files = {
-      whats_new: File.join(PROJECT_FOLDER,  "/Simplenote/Resources/release_notes.txt"),
-      app_store_subtitle: File.join(source_metadata_folder, "subtitle.txt"),
-      app_store_desc: File.join(source_metadata_folder, "description.txt"),
-      app_store_keywords: File.join(source_metadata_folder, "keywords.txt")
-    }
-
-    ios_update_metadata_source(po_file_path: File.join(PROJECT_FOLDER, "/Simplenote/Resources/AppStoreStrings.pot"),
-      source_files: files,
-      release_version: options[:version])
-  end
+  generate_strings_file_for_glotpress
+end
```

Comment:
❔  I'm surprised there's no `git push` happening during this `code_freeze` lane anywhere?
(Or maybe that's hidden within one of the actions that are called here? I'm assuming that's the case, otherwise if we never `git push`'d during code_freeze (even in the old code before this PR) this would never have worked…)

Reply:
I'm not sure how it worked before. I had to manually add a `push_to_git_remote(tags: false)` when I did the 2.20 code freeze. So that'll be here once the `release/2.20` branch is merged to `trunk`.